src/plugins/unified_search/public/filter_bar/filter_item/filter_items.tsx

```diff
@@ -84,6 +84,7 @@ const FilterItemsUI = React.memo(function FilterItemsUI(props: FilterItemsProps)
           readOnly={readOnly}
           suggestionsAbstraction={props.suggestionsAbstraction}
           filtersCount={props.filters.length}
+          dataViews={data?.dataViews}
```

Comment:
depending on the context relying on the data service cause functional test errors, same issue I encountered using the dataViews service directly.